README.md

```diff
@@ -315,3 +315,55 @@ Looking for more guidance? Full documentation for Faststore lives [on this GitHu
 ## ⚡ Performance & QA
 
 This project has strict performance budgets. Right out of the box, this project performs around 95 on Google's Page Speed Insights website, which usually is way more strict than your laptop's chrome lighthouse. Every time you commit to the repository, our QA bots will run and evaluate your code quality. We recommend you NEVER put in production a code that breaks any of the bots. If a bot breaks and still you need to put the code into production, change the bot config (`lighthouserc.js`, `cypress.json`) to make it pass and merge. This way you ensure your website will keep performing well during the years to come.
+
+
+## Adding analytics systems
+
+To boost the performance of the store pages, this store transfers all computation related to analytics to the backend. This backend work as a hub that receives events from the store page and forwards the event to the analytics engine (e.g: google analytics, facebook analytics, google tag manager).
+
+The analytics backend code is inside the `src/api/analytics.ts`.
```

Comment:
I don't know if we should include this piece of info. It may change in the feature and if there's anything worst than no docs, this thing is wrong docs. I don't think people will remember to update this.